Essentials/src/main/java/com/earth2me/essentials/MetaItemStack.java

```diff
@@ -216,11 +215,20 @@ public void addStringMeta(final CommandSource sender, final boolean allowUnsafe,
             } else {
                 throw new Exception(tl("onlyPlayerSkulls"));
             }
-        } else if (split.length > 1 && split[0].equalsIgnoreCase("book") && MaterialUtil.isEditableBook(stack.getType()) && (hasMetaPermission(sender, "book",true, true, ess) || hasMetaPermission(sender, "chapter-" + split[1].toLowerCase(Locale.ENGLISH), true, true, ess))) {
+        } else if (split.length > 1 && split[0].equalsIgnoreCase("book") && MaterialUtil.isEditableBook(stack.getType()) && (hasMetaPermission(sender, "book", true, true, ess) || hasMetaPermission(sender, "chapter-" + split[1].toLowerCase(Locale.ENGLISH), true, true, ess))) {
             final BookMeta meta = (BookMeta) stack.getItemMeta();
             final IText input = new BookInput("book", true, ess);
             final BookPager pager = new BookPager(input);
 
+            // This fix only applies to written books.
+            if (stack.getType() == WRITTEN_BOOK) {
+                // Written books require an author and a title. https://bugs.mojang.com/browse/MC-59153
+                meta.setAuthor(DEFAULT_BOOK_AUTHOR);
```

Comment:
We should probably try to use author meta here if applicable, and only default when it is not available. See the Essentials Item Meta format for books [here](https://wiki.mc-ess.net/wiki/Item_Meta#Books) on the old wiki.